webpack/webpack.dev-stage.config.js

```diff
@@ -0,0 +1,142 @@
+// This is the dev Webpack config. All settings here should prefer a fast build
```

Comment:
I'd suggest using Merge.smart with the dev config here to remove all the duplicate code.  That way this file can basically be overriding the Environment plugin and devServer.